draft/2022-01-26-this-week-in-rust.md

```diff
@@ -187,6 +187,10 @@ Email the [Rust Community Team][community] for access.
 
 * [Senior Frontend Engineer - Kollider, Remote](https://kollider.homerun.co/senior-frontend-engineer/en?)
 
+**Tempus Ex**
+
+* [Several full-time Rust positions available (San Francisco, Atlanta, Austin, and Remote)](https://tempus-ex.com/careers?r=twir)
```

Comment:
Thanks for submitting this link. I strongly prefer (and our audience prefers) not to feature links that include tracking tags like this one. I will include the link, but I am removing the "?r=twir" param.

Reply:
Fair enough. 👍  I'll keep this in mind for future PRs. Thanks!

Reply:
Yw!